tasks/mysql_secure_installation.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: Install python-mysqldb for Ansible
+  package: pkg=python-mysqldb state=present
+
+- name: root password is present
+  mysql_user: name=root host={{ item }} password={{ mysql_root_password }} state=present
+  with_items:
+    - '::1'
+    - '127.0.0.1'
+    - 'localhost'
+
+- name: root has .my.cnf
+  template: src=my.cnf.j2 dest=/root/.my.cnf 
+            owner=root group=root mode=0600
+  tags: my_cnf
+
+# Can use only if ansible ver => 2.1
+#- name: anonymous users are absent
+#  mysql_user: name='' state=absent host_all=yes
+#  when: mysql_remove_anonymous_users
+- name: anonymous users are absent
+  command: 'mysql -ne "{{ item }}"'
+  with_items:
+    - DELETE FROM mysql.user WHERE User=''
+  when: mysql_remove_anonymous_users
+  changed_when: false
+
+- name: remote root login is restricted
+  command: 'mysql -ne "{{ item }}"'
+  with_items:
+    - DELETE FROM mysql.user WHERE User='root' AND Host NOT IN ('localhost', '127.0.0.1', '::1')
+  when: not mysql_allow_remote_root
+  changed_when: false
+
+- name: test database is absent
+  mysql_db: name=test state=absent
+  when: mysql_remove_test_database
+
+- name: access to test database is absent
+  command: 'mysql -ne "{{ item }}"'
+  with_items:
+    - DELETE FROM mysql.db WHERE Db='test' OR Db='test\\_%'
```

Comment:
I'm not sure this even required after `mysql_db: name=test state=absent`. Volunteer is needed to check)